### PR TITLE
Implement New Token Page Chart

### PR DESCRIPTION
**Pull request by math-marcellino**
1. Change timeframe for all charts from `(1D, 1W, 2W, 1M, 4M)` to `(1D, 1W, 2W, 3W, 4W)`.
2. Implement UI for new token page chart (currently using data from legacy token)
3. Store all new token logo images

NOTE: Currently this implemented feature can only be seen via storybook
https://0d4f7a55.risedle-storybook.pages.dev/?path=/story/risedle-v1-flt-page-leveraged-token-chart--leveraged-token-chart

### CODE REVIEW

**Comment by math-marcellino** [comment]
Btw somehow the data fetch in the chart on storybook is blocked by the headers policy I think(?)

image

**Comment by asaadam** [comment]
> Btw somehow the data fetch in the chart on storybook is blocked by the headers policy I think(?)
> 
> image

yap it's blocked by cors. 
backend must allowed from that domain

File: modules/fltPage/components/PriceInfoCard/PriceInfoCardContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+    const resetCurrentNav = () => {
+        setCurrentNav(latestNav);
+        setCurrentNavChange(latestNavChange);
+        setCurrentNavPercentChange(latestPercentChange);
+        setCurrentDate("");
+    };
+
+    // Listen to timeframe change (this is temporary & subject to changes until real data is ready)
+    useEffect(() => {
```
Comment (asaadam) [inline modules/fltPage/components/PriceInfoCard/PriceInfoCardContainer.tsx:56]: please insert your data to dependencies list

then convert your code to something like this

```
useEffect(()=>{
if (data){
switch (activeTimeFrame){
            case Timeframe.Daily: setSelectedData(data.daily) break;
....

}
}
},[...yourDependencies])

File: modules/fltPage/components/PriceInfoCard/PriceInfoCardHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import ArbitrumFlatIcon from "../../../../uikit/icon/network/ArbitrumFlatIcon";
+import BSCFlatIcon from "../../../../uikit/icon/network/BSCFlatIcon";
+
+/**
+ * PriceInfoCardHeaderProps is a React Component properties that passed to React Component PriceInfoCardHeader
+ */
+type PriceInfoCardHeaderProps = {
+    subtitle: string;
+    title: string;
+    chainName: "arbitrum" | "binance";
```
Comment (asaadam) [inline modules/fltPage/components/PriceInfoCard/PriceInfoCardHeader.tsx]: create seperate types for this enum 
so we can re use it later

**Review by asaadam — CHANGES_REQUESTED** [review]

**Comment by asaadam** [comment]
LGTM

**Review by asaadam — APPROVED** [review]

**Pull request merged**